Add instruction for SkyPilot (#1)
Michaelvll/sglang: README.md

1. Copy the [compose.yml](./docker/compose.yaml) to your local machine
2. Execute the command `docker compose up -d` in your terminal.

> ### Method 5: Run on Clouds with SkyPilot

Reviewer: ```suggestion
### Method 5: Run on Kubernetes or Cloud VMs with SkyPilot
```

Author: "Run on Cloud VMs" feels like that we need to run on existing cloud VMs. How about just `Run on Kubernetes or Cloud with SkyPilot`?

Reviewer: Sg, plural? `Run on Kubernetes or Clouds with SkyPilot`

> To deploy on any cloud or Kubernetes cluster, you can use [SkyPilot](https://github.com/skypilot-org/skypilot).

Reviewer: L92:94: swap K8s and cloud?

> 1. Install SkyPilot and setup your cloud or Kubernetes cluster: see [SkyPilot's documentation](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html).

Reviewer: ```suggestion
1. Install SkyPilot and set up cloud VM or Kubernetes cluster access: see [SkyPilot's documentation](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html).
```

2. Deploy on your own infra with a single command and get the HTTP API endpoint:
<details>
<summary>SkyPilot YAML: <code>sglang.yaml</code></summary>

```yaml
# sglang.yaml
envs:
  HF_TOKEN: null

resources:
  image_id: docker:lmsysorg/sglang:latest
  accelerators: A100
  ports: 30000

run: |
  conda deactivate
  python3 -m sglang.launch_server \
    --model-path meta-llama/Meta-Llama-3.1-8B-Instruct \
    --host 0.0.0.0 \
    --port 30000
```

</details>

```bash

> # Deploy on any cloud or Kubernetes cluster, use --cloud <cloud> to select a specific cloud provider.

Reviewer: ```suggestion
# Deploy on any cloud or Kubernetes cluster. Use --cloud <cloud> to select a specific cloud provider.
```
